### PR TITLE
WIP: Remove app header

### PR DESCRIPTION
**Pull request by thevoiceofzeke**
*Note: I made a mess of this during rebasing. See #684*

### CODE REVIEW

File: components/staticFeeds/sample-messages.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
       "goLiveDate": "2017-08-03",
       "expireDate": null,
       "featureImageUrl": null,
-      "priority": "high",
+      "priority": null,
```
Comment (apetro) [inline components/staticFeeds/sample-messages.json]: Turning off high priority here inconsistent with `title` of this sample notification.

**Comment by vertein** [comment]
update changelog to reflect changes

File: components/portal/misc/partials/toast-add-to-home.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+-->
+<md-toast>
+  <div class="md-toast-content">
+    <span class="md-toast-text" flex>Add to home?</span>
+    <div layout="row">
+      <md-button ng-click="addToHome()"
+                 class="md-raised md-accent">
+        <span ng-if="!successfullyAdded && !addToHomeFailed && !savingAddToHome"><md-icon>add</md-icon> Add</span>
+        <span ng-if="!successfullyAdded && !addToHomeFailed && savingAddToHome" ng-disabled="true">
+        <i class="fa fa-fw fa-spin fa-circle-o-notch" ng-show="savingAddToHome"></i>
```
Comment (ChristianMurphy) [inline components/portal/misc/partials/toast-add-to-home.html]: :art: indent one level more than line 28

File: components/portal/misc/partials/toast-add-to-home.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      </span>
```
Comment (ChristianMurphy) [inline components/portal/misc/partials/toast-add-to-home.html]: :art: indent to match line 28

File: components/portal/main/controllers.js [hunk 4]
```diff
@@ -39,7 +39,11 @@ define(['angular', 'require'], function(angular, require) {
      * sets the document title to that value.
      * @param {string} [pageTitle] - Name of specific page viewed.
      */
+<<<<<<< HEAD
```
Comment (ChristianMurphy) [inline components/portal/main/controllers.js]: merge conflict snuck in

**Comment by thevoiceofzeke** [comment]
Closing this because I fumbled the bar during rebase and it retracing my steps the hard way for a new PR was easier than untangling that knot.

**Pull request closed without merging**